cvat/apps/quality_control/rules/conflicts.rego

```diff
@@ -58,28 +57,6 @@ allow if {
     organizations.is_member
 }
 
-allow if {
-    input.scope == utils.VIEW
-    utils.is_sandbox
-    quality_utils.is_task_staff(input.resource.task, input.resource.project, input.auth)
-    utils.has_perm(utils.WORKER)
-}
-
-allow if {
-    input.scope == utils.VIEW
-    input.auth.organization.id == input.resource.organization.id
-    utils.has_perm(utils.USER)
-    organizations.has_perm(organizations.MAINTAINER)
-}
-
-allow if {
-    input.scope == utils.VIEW
-    quality_utils.is_task_staff(input.resource.task, input.resource.project, input.auth)
-    input.auth.organization.id == input.resource.organization.id
-    utils.has_perm(utils.WORKER)
-    organizations.has_perm(organizations.WORKER)
-}
-
```

Comment:
@zhiltsov-max, I've not found any place where these rules are used. Could you let me know why you added them? Are you going to introduce the GET `/api/quality/conflicts/<id>` endpoint?